jnigen/lib/src/elements/elements.dart

```diff
@@ -97,6 +97,12 @@ class ClassDecl extends ClassMember implements Element<ClassDecl> {
 
   String get internalName => binaryName.replaceAll(".", "/");
 
+  /// The number of super classes this type has.
+  ///
+  /// Will be populated by [Linker].
```

Comment:
Will be populated by -> Populated by

(e.g. the comment should be true when reading it in arbitrary points in the timeline of our pipeline.)

Idem below.